### PR TITLE
Add helm broker chart

### PR DESCRIPTION
**Pull request by jasiu001**
Related to [#5213](https://github.com/kyma-project/kyma/issues/5213)

### CODE REVIEW

File: charts/helm-broker/values.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  # etcd port uses by helm broker
+  etcdClientPort: 2379
+  # temporary directory size limit uses in broker/controller containers
+  tmpDirSizeLimit: 1Gi
+
+broker:
+  tillerTLSInsecure: true
+
+ctrl:
+  # enable/disable documentation component for helm broker addons
```
Comment (polskikiel) [inline charts/helm-broker/values.yaml]: i would write `enable/disable docs topics support for helm controller` :p

File: charts/helm-broker/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+name: helm-broker
+description: Helm service-broker deployment chart.
```
Comment (polskikiel) [inline charts/helm-broker/Chart.yaml]:
```suggestion
description: Helm broker deployment chart.
```

File: charts/helm-broker/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Helm Broker
+
+## Overview
+
+Helm Broker provides addons in the [Service Catalog](https://github.com/kubernetes-sigs/service-catalog). An addon is an abstraction layer over a Helm chart which enables you to provide more information about the Helm chart. For example, an addon can provide plan definitions or binding details. Service Catalog requires this information. Addons are services available in Service Catalog.
```
Comment (klaudiagrz) [inline charts/helm-broker/README.md]:
```suggestion
The Helm Broker is a [Service Broker](https://kyma-project.io/docs/master/components/service-catalog/#service-brokers-overview) which exposes Helm charts as Service Classes in the [Service Catalog](https://kyma-project.io/docs/master/components/service-catalog/#overview-overview). To do so, the Helm Broker uses the concept of addons. An addon is an abstraction layer over a Helm chart which provides all information required to convert the chart into a Service Class.
```

File: charts/helm-broker/README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+
+Helm Broker implements the [Service Broker API](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md). For more information about the Service Brokers, see the [Service Brokers overview](https://github.com/kyma-project/kyma/blob/master/docs/helm-broker/03-01-create-addons.md) documentation. Find the details about the Helm Broker in the [docs](https://github.com/kyma-project/kyma/tree/master/docs/helm-broker) repository.
```
Comment (klaudiagrz) [inline charts/helm-broker/README.md]:
```suggestion
The Helm Broker implements the [Open Service Broker API](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md). For more information about the  Helm Broker, read the [documentation](https://kyma-project.io/docs/master/components/helm-broker/).
```

File: charts/helm-broker/README.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# Helm Broker
+
+## Overview
+
+The Helm Broker is a [Service Broker](https://kyma-project.io/docs/master/components/service-catalog/#service-brokers-overview) which exposes Helm charts as Service Classes in the [Service Catalog](https://kyma-project.io/docs/master/components/service-catalog/#overview-overview). To do so, the Helm Broker uses the concept of addons. An addon is an abstraction layer over a Helm chart which provides all information required to convert the chart into a Service Class.
+
+Helm Broker implements the [Service Broker API](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md). For more information about the Service Brokers, see the [Service Brokers overview](https://github.com/kyma-project/kyma/blob/master/docs/helm-broker/03-01-create-addons.md) documentation. Find the details about the Helm Broker in the [docs](https://github.com/kyma-project/kyma/tree/master/docs/helm-broker) repository.
```
Comment (klaudiagrz) [inline charts/helm-broker/README.md]:
```suggestion
The Helm Broker implements the [Open Service Broker API](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md). For more information about the  Helm Broker, read the [documentation](https://kyma-project.io/docs/master/components/helm-broker/).
```

File: charts/helm-broker/README.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# Helm Broker
+
+## Overview
+
+The Helm Broker is a [Service Broker](https://kyma-project.io/docs/master/components/service-catalog/#service-brokers-overview) which exposes Helm charts as Service Classes in the [Service Catalog](https://kyma-project.io/docs/master/components/service-catalog/#overview-overview). To do so, the Helm Broker uses the concept of addons. An addon is an abstraction layer over a Helm chart which provides all information required to convert the chart into a Service Class.
+
+The Helm Broker implements the [Open Service Broker API](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md). For more information about the  Helm Broker, read the [documentation](https://kyma-project.io/docs/master/components/helm-broker/).
```
Comment (klaudiagrz) [inline charts/helm-broker/README.md]:
```suggestion
The Helm Broker implements the [Open Service Broker API](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md). For more information about the Helm Broker, read the [documentation](https://kyma-project.io/docs/master/components/helm-broker/).
```

**Pull request merged**